crates/bevy_render/src/settings.rs

```diff
@@ -93,6 +93,27 @@ impl Default for WgpuSettings {
     }
 }
 
+/// An enum describing how the renderer will initialize resources.
```

Comment:
It might be nice to link back to `RenderPlugin` here, to better explain how this could be used.